applications/GeoMechanicsApplication/custom_utilities/stress_strain_utilities.cpp

```diff
@@ -176,4 +177,71 @@ std::vector<Vector> StressStrainUtilities::CalculateStrains(const std::vector<Ma
     return result;
 }
 
+void StressStrainUtilities::CalculatePrincipalStresses(const Vector& rCauchyStressVector,
+                                                       Vector&       rPrincipalStressVector,
+                                                       Matrix&       rEigenVectorsMatrix)
+{
+    auto   stress_tensor = MathUtils<double>::StressVectorToTensor(rCauchyStressVector);
+    Matrix PrincipalStressMatrix;
+    Matrix EigenVectorsMatrix;
```

Comment:
These names of local variables don't comply with the Kratos Style Guide:
```suggestion
    Matrix principal_stress_tensor;
    Matrix eigen_vectors;
```